src/software/simulated_tests/friendly_goal_scored.py

```diff
@@ -0,0 +1,51 @@
+import software.python_bindings as tbots
+from proto.import_all_protos import *
+
+from software.simulated_tests.validation import (
+    Validation,
+    create_validation_geometry,
+    create_validation_types,
+)
+
+
+class FriendlyGoalScored(Validation):
```

Comment:
nit: The name here can be misinterpreted to mean that the ball was scored in the friendly goal. Could be updated to something like `FriendlyTeamScored`. Same with the 4 validation functions

Reply:
fixed

Reply:
The class + file name still need to be updated